api.ts

```diff
@@ -247,6 +247,32 @@ export class MissingScopesError extends Error {
   }
 }
 
+/**
+ * An error that will be thrown by {@link Fetcher.fetch} when the response body from the external system
+ * exceeds packs platform limits
+ *
+ * This error can be caught and retried by requesting less data from the external system through
+ * a smaller page size or omitting large fields.
+ *
+ * @hidden
+ */
+export class ResponseSizeTooLargeError extends Error {
```

Comment:
RequestTooLarge would also be good to add at some point, but doesn't need to block this.

Reply:
Will add as needed. Starting with this. Once this is going in a real pack can add the other error as well will be very similar